mkdocs.yml

```diff
@@ -6,6 +6,15 @@ plugins:
         on_pre_build: "docs.hooks:pre_build"
 repo_url: 'https://github.com/aeternity/aepp-sdk-js'
 
+theme:
+  name: readthedocs
```

Comment:
didn't you notice that the navigation is broken when using this theme? Or it is already fixed somehow? https://github.com/aeternity/aepp-sdk-js/pull/1077#discussion_r514587519

Reply:
I'm writing an issue on this right now, thing is that the default theme is very ugly and unusable, and people participating in a currently ongoing hackathon mind that more than the buggy nav 😄 I'm proposing solutions, hang in there.

Reply:
I think the broken navigation worse than a theme that doesn't look enough nice 🤷‍♀️